DMRIModuleTemplates/Tractography_Scripted/TemplateKey.py

```diff
@@ -128,38 +145,24 @@ def hasImageData(self,volumeNode):
     return True
 
 
-  def run(self, inputFB):
+  def run(self, inputFB, outputFB):
     """
     Run the actual algorithm
     """
 
     logging.info('Processing started')
 
     assert inputFB and inputFB.IsA("vtkMRMLFiberBundleNode")
-    if not inputFB.GetPolyData():
+    assert outputFB and outputFB.IsA("vtkMRMLFiberBundleNode")
+    if not inputFB.GetPolyData() or not outputFB.GetPolyData():
       logging.warn("Missing PolyData")
 
-    # get proxy object to access underlying array
-    fb_numpy = dsa.WrapDataObject(inputFB.GetPolyData())
-
-    # get first tensor key
-    ten_key = next(ifilter(lambda x: "Tensor_" in x, fb_numpy.PointData.keys()), None)
```

Comment:
@zhangfanmark did you intend to get rid of the part that is doing the example calculation here?